tests/acceptance/ReinstallFromScratchCest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+use MailPoet\Models\Setting;
+use MailPoet\Test\DataFactories\Form;
+use MailPoet\Test\DataFactories\Newsletter;
+use MailPoet\Test\DataFactories\Segment;
+use MailPoet\Test\DataFactories\Subscriber;
+use PHPUnit_Framework_Assert as Asserts;
+
+require_once __DIR__ . '/../DataFactories/Form.php';
+require_once __DIR__ . '/../DataFactories/Newsletter.php';
+require_once __DIR__ . '/../DataFactories/Segment.php';
+require_once __DIR__ . '/../DataFactories/Subscriber.php';
+
+// This test should run last, because it removes all test data
```

Comment:
Is this comment still true? I am looking at `_after` and it seems that test data are being repopulated at the end.